website/scripts/index.js

```diff
@@ -43,3 +43,6 @@ function showSlides(n) {
 	dots[slideIndex-1].className += " active";
 	timeOut = setTimeout(() => plusSlides(1), 5000); //Increments slide every 5 seconds automatically, timer is reset whenever a slide is manually selected
 }
+
+	//Functions for the login page
+	
```

Comment:
This is something I need to change about the other pages. The plan with the scripts folder is to treat it like the styles folder (so there will be a unified script for the header functions, and then each page would make it's own script if it needs it). I'll make those changes soon, just wanted to let you know.